src/client/App.jsx

```diff
@@ -33,7 +33,7 @@ const App = () => {
         const fetchedUser = async () => {
             try {
                 const response = await fetch(`${USER_URL.USER_ROOT}${id}`, {
-                    method: 'GET',
+                    method: `${FETCH_METHOD.GET}`,
```

Comment:
I like the approach here but try not to confuse `fetch` the function with a HTTP Request: `HTTP_METHOD.GET` may be a better fit